tests/test_concurrency.py

```
@@ -66,16 +66,20 @@ def wrapped(index: int) -> None:
 def test_each_thread_gets_its_own_connection(tmp_path):
     """אותו אובייקט Database, חיבור אחר בכל תהליכון."""
     db = connect(tmp_path / "t.db")
-    seen: dict[str, int] = {}
+    # שומרים את אובייקט החיבור עצמו ולא רק את `id()`: ‏id של אובייקט
+    # שנאסף לזבל **ממוחזר**, ולכן תהליכון שסיים יכול לתרום id שחיבור
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)